LiverMarkups/MRML/vtkMRMLMarkupsBezierSurfaceDisplayNode.cxx

```diff
@@ -50,7 +50,7 @@ vtkMRMLNodeNewMacro(vtkMRMLMarkupsBezierSurfaceDisplayNode);
 vtkMRMLMarkupsBezierSurfaceDisplayNode::vtkMRMLMarkupsBezierSurfaceDisplayNode()
   :ClipOut(false), WidgetVisibility(true),
    ResectionMarginColor{0.0f, 0.0f, 0.0f}, UncertaintyMarginColor{0.0f, 0.0f, 0.0f},
-   ResectionOpacity(1.0f), GridDivisions(0.0f), GridThickness(0.0f), ShowResection2D(false), EnableFlexibleBoundary(false), MirrorDisplay(false),
```

Comment:
Divisions should be initialized as `0`, instead of  `0.0` the underlying datatype is `unsigned int`